lib/main.js

```diff
@@ -3,96 +3,178 @@
 const url = require('url');
 const JSONStream = require('JSONStream');
 const request = require('request');
-const Writer = require('asset-pipe-js-writer');
+const JsWriter = require('asset-pipe-js-writer');
+const CssWriter = require('asset-pipe-css-writer');
+const assert = require('assert');
+const { extname } = require('path');
+const { Stream } = require('stream');
+
+const extensions = Object.freeze({
+    JS: '.js',
+    CSS: '.css',
+});
+
+const endpoints = Object.freeze({
+    UPLOAD: 'feed',
+    BUNDLE: 'bundle',
+});
+
+function post(options) {
+    const isStream = options.body instanceof Stream;
```

Comment:
https://www.npmjs.com/package/is-stream?

Reply:
instead of coupling to node core streams

Reply:
nice